io/net/Network_config.py

```diff
@@ -110,7 +110,8 @@ def test_networkconfig(self):
                     self.fail("mismatch in mtu")
             if 'MTU' in val:
                 cmd, mtu_val = val.split('MTU:')
-                mtu_val, cmd = mtu_val.split(' ')
+                mtu_val, cmd = mtu_val.split(' M')
```

Comment:
Can you explain with example why this was not working before and why it would work now please